src/parser/md.rs

```diff
@@ -129,7 +129,9 @@ Please comment out option `comments_as_aside` until the next version, and rename
                     .unwrap_or(&trans);
 
                 let mut path = PathBuf::from(into.parent().unwrap_or_else(|| Path::new(".")));
-                path.push(target);
+
+                let sub_path = PathBuf::from(target);
```

Comment:
Why do we need to allocate a temporary `PathBuf` if we are appending this to another `PathBuf` anyway?